web/packages/core/src/polyfills.js

```diff
@@ -163,42 +163,16 @@ function polyfill_dynamic_frames() {
     observer.observe(document, { childList: true, subtree: true });
 }
 
-function polyfill_frames() {
+function polyfill_everything() {
+    falsify_plugin_detection();
+    polyfill_static_content();
+    polyfill_dynamic_content();
     polyfill_static_frames();
-    if (running_polyfills.indexOf("dynamic-content") != -1) {
-        polyfill_dynamic_frames();
-    }
+    polyfill_dynamic_frames();
 }
-let running_polyfills = [];
-let polyfills = {
-    "static-content": polyfill_static_content,
-    "dynamic-content": polyfill_dynamic_content,
-    "plugin-detect": falsify_plugin_detection,
-    frames: polyfill_frames,
-};
-
-exports.polyfill = function polyfill(polyfill_list) {
-    for (let i = 0; i < polyfill_list.length; i += 1) {
-        if (running_polyfills.indexOf(polyfill_list[i]) !== -1) {
-            continue;
-        }
-
-        if (
-            !Object.prototype.hasOwnProperty.call(polyfills, polyfill_list[i])
-        ) {
-            throw new Error(
-                "Requested nonexistent polyfill: " + polyfill_list[i]
-            );
-        }
-
-        running_polyfills.push(polyfill_list[i]);
-
-        let this_polyfill = polyfills[polyfill_list[i]];
-
-        if (this_polyfill.dependencies !== undefined) {
-            polyfill(this_polyfill.dependencies);
-        }
 
-        this_polyfill();
+exports.polyfill = function polyfill(polyfill) {
+    if (polyfill) {
+        polyfill_everything();
     }
 };
```

Comment:
Since this does nothing in the case of `polyfill(false)`, let's make this `polyfill()` with no arguments, copy `polyfill_everything` into here, and then remove `polyfill_everything`